homeassistant/components/wyoming/tts.py

```diff
@@ -143,7 +147,7 @@ async def async_get_tts_audio(self, message, language, options):
         except (OSError, WyomingError):
             return (None, None)
 
-        if options[tts.ATTR_AUDIO_OUTPUT] == "wav":
+        if options.get(tts.ATTR_AUDIO_OUTPUT) != "raw":
```

Comment:
do we still need to guard for raw ?

Reply:
Yes, until https://github.com/home-assistant/core/pull/102869 is merged